conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -29,6 +29,7 @@ CONDARC
 cat << EOF | docker run -i \
                         -v ${RECIPE_ROOT}:/recipe_root \
                         -v ${FEEDSTOCK_ROOT}:/feedstock_root \
```

Comment:
Both of these should probably be in the user's home directory or at least not `/`. Not sure exactly how we will accomplish that as we create the user's home directory after. Any thoughts on this? I'll try to think about it some as well.